capstone-frontend/src/components/BookSearchList.jsx

```diff
@@ -7,26 +7,51 @@ class BookSearchList extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      bookList: []
+      bookSearchList: []
     }
-    this.getData = this.getData.bind(this);
   }
 
-  getData() {
+  getData = () => {
     fetch(`/api/search?searchTerm=${this.props.searchQuery}`)
       .then(response => response.json())
-      .then(res => this.setState({ bookList: res }))
+      .then(res => this.setState({ bookSearchList: res }))
       .catch(err => console.log(err));
   }
 
+  getUserBookLists = () => {
```

Comment:
nit: Same as the helper functions I commented on in BookSearchTileTest - return the object literal directly for these.

I would also add a TODO with a linked issue number to replace these with actual API calls once the API is implemented.

Reply:
Added!